direction/direction.go

```diff
@@ -1,8 +1,11 @@
 package direction
 
+// Direction represents direction in which migration should be applied
+type Direction string
```

Comment:
why string? why not bool?

Reply:
@andreigornostajev because there is not need for this:
https://github.com/wallester/migrate/pull/11/files#diff-3a6a81f23576f46a4c6e6ab834f04b16L63

Reply:
The string type looks confusing to me too, as the migrations can be only be up or down.

Reply:
I still like more boolean type

Reply:
reverted